cs/build/nuget/Common.targets

```diff
@@ -127,6 +127,12 @@
         <DependentUpon>%(BondCodegen.Identity)</DependentUpon>
       </_BondGeneratedFiles>
 
+      <_BondGeneratedFiles Include="@(BondCodegen -> '$(BondOutputDirectory)\%(FileName)_grpc.cs')"
+                           Condition="$([System.String]::new('%(BondCodegen.Options)').Contains('--grpc'))">
```

Comment:
This also needs to handle the case when the property `$(BondOptions)` contains "--grpc". Note that `$(BondOptions)` is only used for [`BondCodegen` items that don't have overridden Options metadata](https://github.com/Microsoft/bond/blob/ca0d1c63da2760ad5753a6a8bf29b00f8cb01585/cs/build/nuget/Common.targets#L103-L104). One of the examples can be changed to use `$(BondOptions)` as a test for this behavior.